packages/nimble-components/src/table-column/date-text/tests/table-column-date-text.spec.ts

```diff
@@ -78,32 +78,41 @@ describe('TableColumnDateText', () => {
             expect(column.checkValidity()).toBeTrue();
         });
 
-        // WebKit skipped, see https://github.com/ni/nimble/issues/1940
-        it('changing fieldName updates display #SkipWebkit', async () => {
+        it('changing fieldName updates display', async () => {
+            const fieldValue = new Date('Dec 10, 2012, 10:35:05 PM').valueOf();
+            const anotherFieldValue = new Date(
```

Comment:
nit: can we use the names `expected` and `actual` for these where appropriate? Looks like we don't tend to save actual to named variables which is fine, but can we name these `expected` value vars as such, i.e. `expectedFieldValue`?